lib/socialcast-git-extensions/github.rb

```diff
@@ -19,8 +19,10 @@ def authorization_token
         username = current_user
         raise "Github user not configured.  Run: `git config --global github.user 'me@email.com'`" if username.empty?
         password = HighLine.new.ask("Github password for #{username}: ") { |q| q.echo = false }
+        token_name = HighLine.new.ask('Github token name for this machine')
+        token_name = 'Socialcast Git eXtension' if token_name.empty?
```

Comment:
I think it'd be helpful to have the prompt in the above line display this default value, e.g. something like
```ruby
default_token_name = 'Socialcast Git eXtension'
token_name = HighLine.new.ask("Github token name for this machine (default: '#{default_token_name}'):")
token_name = default_token_name if token_name.empty?
```